conda_gitenv/deploy.py

```diff
@@ -54,6 +54,11 @@ def deploy_tag(repo, tag_name, target):
 
 
 def create_env(pkgs, target, pkg_cache):
+    # Empty package cache so that it will reinitialised.
+    conda.install.package_cache_ = {}
```

Comment:
@lbdreyer there you go, doesn't `package_cache()` use and set `package_cache_` under the hood?